prime-dotnet-webapi/Services/VerifiableCredentialService.cs

```diff
@@ -19,6 +19,7 @@ public class WebhookTopic
         public const string Connections = "connections";
         public const string IssueCredential = "issue_credential";
         public const string RevocationRegistry = "revocation_registry";
+        public const string BasicMessage = "basicmessages";
```

Comment:
For consistency with the other this could be snake_cased

Reply:
Yeah I expected it to be snake cased, but that is what they send us